docs/interfaces/mx4/leds.md

```diff
@@ -1 +1,187 @@
-On the MX-4 platform the LEDs are configured under `/opt/hm/pic_attributes` 
+---
+title: LEDs (MX-4)
+tags:
+  - LED
+  - C61
+  - MX-4
+  - T30
+  - T30 FR
+  - CT
+---
+
+## Overview
+
+On the MX-4 platform the LEDs are configured under `/opt/hm/pic_attributes/LED` for complete list of [pic_attributes](pic_attributes.md)
+
+All leds except the power led can be controlled from linux see [Pre configured LED behavior](#Pre configured LED behavior).
+
+## WiFi LEDs
+
+The green wifi LED (mx4-wifi) is setup to be lit on netdev. you can overwrite this by set it to none or another trigger. Some platform does not have the mx4-wifi-red and this led is free to be used.
+```
+echo none > /sys/class/leds/mx4-wifi/trigger
+# On
+echo 1 >  /sys/class/leds/mx4-wifi/brightness
+echo 1 >  /sys/class/leds/mx4-wifi-red/brightness
+# Off
+echo 0 >  /sys/class/leds/mx4-wifi/brightness
+echo 0 >  /sys/class/leds/mx4-wifi-red/brightness
+```
+
+## Script to set LEDs
+you can use a script under /opt/hm called set_led_flash.sh
+```
+root@mx4-t30-29009999:~# /opt/hm/set_led_flash.sh
+Usage: /opt/hm/set_led_flash.sh <index> <color> <freq>
+
+index = 2 - GSM LED
+index = 3 - GPS LED
+index = 4 - FUNC LED
+index = 5 - BUZZER
+
+color = 0 - led off
+color = 1 - blink off/green
+color = 2 - blink off/orange
+color = 3 - blink green/orange
+
+freq = 0 - no blink
+freq = 1-255 - Flashing with frequency; freq * 100 milliseconds
+```
+### Pre configured LED behavior
+
+#### PWR
+
+The power led is is green or flashing green yellow/orange if charging an internal battery. Depending on the level of charge it will change speed between the flashing green<-->yellow, until it is lit green. If the unit does not have a battery it will continue flash green<-->yellow.
```

Comment:
```suggestion
The power LED is is green or flashing green yellow/orange if charging an internal battery. Depending on the level of charge it will change speed between the flashing green ↔ yellow until it is lit green. If the unit does not have a battery it will continue to flash green ↔ yellow.
```